src/qt/transactionrecord.cpp

```diff
@@ -172,10 +174,11 @@ void TransactionRecord::updateStatus(const interfaces::WalletTxStatus& wtx, int
     status.depth = wtx.depth_in_main_chain;
     status.cur_num_blocks = numBlocks;
 
-    if (!wtx.is_final)
-    {
-        if (wtx.lock_time < LOCKTIME_THRESHOLD)
-        {
+    QDateTime currentDate = QDateTime::currentDateTime();
+    qint64 secs = blockDate.secsTo(currentDate);
+
+    if ((secs < 90 * 60) && !wtx.is_final) {
```

Comment:
This `90 * 60` is meant to match the `if(secs < 90*60)` in `src/qt/bitcoingui.cpp`, right? If so, I suggest adding a common constant to make sure these values stay in sync over time :-)

Reply:
Nice. Which file is a proper place for this common constant?

Reply:
I don't know the QT code well enough to give a good recommendation here. I'll let other developers fill in :-)

Reply:
What is 90*60? How do you get this number?

Reply:
As @practicalswift mentioned above:

https://github.com/bitcoin/bitcoin/blob/36323e2ac6cca134de47b06c3f6403de91ee9eda/src/qt/bitcoingui.cpp#L838-L842

Reply:
Ref #1026